README.md

```diff
@@ -41,61 +41,60 @@ Killgrave is a simulator for HTTP-based APIs, in simple words a **Mock Server**,
+* Validation of request headers.
+* Using regex to allow different parameters in headers and urls.
+* Custom body and dynamic body responses.
 * Using all content-types bodies, (`application/json`, `text/html`, `text/plain`, `image/jpeg`, etc. )
-* Configure the headers on body responses.
-* Configure your CORS server options.
+* Configure response headers.
+* Configure CORS.
 * Simulate network issues and server high loads with imposter responses delay.
 * Using configurable proxy server to call to the original server.
-* Run the tool using flag or using a config file.
-* Run your mock server using a watcher to not reload on each change.
+* Run the tool using flags or using a config file.
+* Run your mock server using a watcher to reload on configuration changes.
 
 ## Concepts
 
 ### Imposters
-Maybe the most common concept inside of Killgrave tool is the `imposters`. Imposters are the files that we will use to configure
-how we want that our server respond.
 
-You can identify an `imposter` file on Killgrave because they must have the extension `.imp.json`.
+Imposters are the most important concept of the Killgrave tool. They define the rules that determine how the server should respond to a request.
 
-Yo can learn more about how to configure the imposters on the [Imposter Configuration Section](#imposter).
+You can identify a Killgrave imposter file by its extension: `.imp.json`.
+
+You can learn more about how to configure imposters in the [Imposter Configuration Section](#imposter).
 
 ## Installing
-> :warning:  Killgrave is a very robust tool and is using on some companies even on production environments, but we don't have a
-version 1 for today. For that Killgrave works using [`SemVer`](https://semver.org/) but in 0 version, which means that the 'minor' will be changed when some broken changes are introduced into the application, and the 'patch' will be changed when a new feature with new changes is added or for bug fixing. As soon as v1.0.0 be released, `Killgrave` will start to use [`SemVer`](https://semver.org/) as usual.
+> :warning:  Even though Killgrave is a very robust tool and is being used by some companies in production environments, it's still in initial development. Therefore, 'minor' version numbers are used to signify breaking changes and 'patch' version numbers are used for non-breaking changes or bugfixing. As soon as v1.0.0 is released, Killgrave will start to use [`SemVer`](https://semver.org/) as usual.
 
-You can install Killgrave on different ways, but all of them are very simple:
+You can install Killgrave in different ways, but all of them are very simple:
 
 ### Go Toolchain
 
-One of them is of course using `go get`, Killgrave is a Go project so could be compiled using the `go toolchain`:
+One of them is of course using `go get`, Killgrave is a Go project and can therefore be compiled using the `go toolchain`:
 
 ```sh
 $ go get -u github.com/friendsofgo/killgrave/cmd/killgrave@{version}
```

Comment:
Could be interesting to take the opportunity of this clean up to mention that since Go v1.16 it's better to use `go install` instead of `go get` for installing binaries from source code.

What do you think about simply swapping both commands? 
I think latest versions of Killgrave require Go v1.16 (because we started to use `goembed`), so I think we can assume the user should have Go v1.16+ installed if wants to install Killgrave from source code.

Reply:
Sounds good! I updated it.